src/MainPage/MainPage.js

```diff
@@ -24,27 +24,20 @@ import volcanoImg from '../Utilities/Images/Box_Volcano_BDSP.png'
 
 function MainPage() {
   const navigate = useNavigate();
-  const params = useParams();
+  const { pokeId } = useParams();
```

Comment:
We ought to make sure this does have any ramifications on other bits of functionality outside this ticket